pygfunction/api.py

```diff
@@ -0,0 +1,39 @@
+from pygfunction.gfunction import gFunction
+from pygfunction.boreholes import Borehole
+from typing import Dict, Optional, Any, List
+
+
+class PYG(object):
```

Comment:
I wonder about renaming the class.  It kinda depends on what we expect this class to ultimately contain/do.  Is it always going to be representative of a single borehole field?  I'm open to suggestion, but this feels a bit vague.